contrib/analyze-local-images/main.go

```diff
@@ -257,7 +258,8 @@ func listenHTTP(path, allowedHost string) {
 
 	restrictedFileServer := func(path, allowedHost string) http.Handler {
 		fc := func(w http.ResponseWriter, r *http.Request) {
-			if r.Host == allowedHost {
+			host, _, err := net.SplitHostPort(r.RemoteAddr)
+			if nil == err && strings.EqualFold(host, allowedHost) {
```

Comment:
Can you reorder this to `if err == nil && ...`? It's idiomatic to write the error check in this order in Go.